docs/what-is-rocm.rst

```diff
@@ -116,6 +116,7 @@ Performance
   ":doc:`Omniperf <omniperf:index>`", "System performance profiling tool for machine learning and HPC workloads"
   ":doc:`Omnitrace <omnitrace:index>`", "Comprehensive profiling and tracing tool for HIP applications"
   ":doc:`ROCm Bandwidth Test <rocm_bandwidth_test:index>`", "Captures the performance characteristics of buffer copying and kernel read/write operations"
+  ":doc:`TransferBench <transferbench:index>`", "Benchmarks simultaneous transfers between user-specified devices"
```

Comment:
This description should match the first line in the docs - or be a closer in terms of details. 
TransferBench is a utility to benchmark simultaneous transfers between user-specified devices (CPUs or GPUs).

Reply:
@lpaoletti I have derived the description from the first line in the doc. I noticed the description of other components in the list start with the utility of the tool hence I kept it in the same format.

Let me know if I should change it to "Utility to benchmark simultaneous transfers between user-specified devices (CPUs or GPUs)".